packages/experimental/RadioGroup/src/Radio/Radio.types.ts

```diff
@@ -38,9 +38,20 @@ export interface RadioTokens extends FontTokens, IColorTokens, IForegroundColorT
    */
   radioBorderWidth?: number;
 
+  /**
+   * The flex direction of the root
+   */
+  flexDirection?: ViewStyle['flexDirection'];
+
+  /**
+   * The flex direction of the root
+   */
+  marginTop?: ViewStyle['marginTop'];
+
   /**
    * States that can be applied to a Radio
    */
+  isBelow?: RadioTokens;
```

Comment:
Can we rename this to be more descriptive of what is below what? like labelPositionBelow?

Reply:
That's a good idea. Replaced!